site/snippets/arcana-auth.ts

```diff
@@ -1,20 +1,19 @@
-import { WalletClientSigner, type SmartAccountSigner } from "@alchemy/aa-core";
-import { AuthProvider } from "@arcana/auth";
-import { createWalletClient, custom } from "viem";
+import { ArcanaAuthSigner } from "@alchemy/aa-signers";
```

Comment:
this needs to change now that we've changed how we import Arcana from aa-signers:
```suggestion
import { ArcanaAuthSigner } from "@alchemy/aa-signers/arcana-auth";
```

Reply:
per your question somewhere, you can test if the imports work on this file with no ts errors after building the packages